### PR TITLE
Core skills screen - Implement screen UI & backend integration

### PR DESCRIPTION
**Pull request by eparpalac**
Demo: https://drive.google.com/file/d/14LM-BuU7k3s88R7_5PD6v_9Ou1lQEoDW/view?usp=sharing
Testing credentials: dude@mail.com/123123123

### CODE REVIEW

File: lib/modules/common/constants.dart [hunk 1]
```diff
@@ -2,4 +2,14 @@ class Paths {
   static const String HOME = '/';
```
Comment (ccosneanu) [inline lib/modules/common/constants.dart:2]: i did refactor it, in `AppRoute`, lets use that.

File: lib/services/auth_service.dart [hunk 2]
```diff
@@ -62,6 +62,8 @@ class AuthService {
   Future<User> me() async {
     final url = BASE_URL + paths[AuthPath.me];
 
+    String authToken = await FlutterKeychain.get(key: 'authToken');
```
Comment (ccosneanu) [inline lib/services/auth_service.dart:65]: Hurray we have persisted our token! ))

**Review by ccosneanu — CHANGES_REQUESTED** [review]

**Comment by ccosneanu** [comment]
Oh, and this is kinda silly, but make sure to have 13 commits.

```zsh
git commit --allow-empty -m "+1 SP"
```

looks weird, but we already went for it.

File: lib/modules/common/constants.dart (hunk 1 above)
Comment (eparpalac) [reply lib/modules/common/constants.dart:2]: ok, sure) I will resolve the conflicts and will change this

**Review by ccosnean — APPROVED** [review]

**Pull request merged**